markdown/renderer.go

```diff
@@ -78,13 +86,15 @@ func (mr *markdownRenderer) RenderNode(w io.Writer, node *blackfriday.Node, ente
 }
 
 func (mr *markdownRenderer) renderChildren(node *blackfriday.Node) []byte {
-	buf := mr.buf
+	oldBuf := mr.buf
 	mr.buf = bytes.NewBuffer(nil)
+	resBuf := bytes.NewBuffer(nil)
 	for n := node.FirstChild; n != nil; n = n.Next {
 		n.Walk(func(node *blackfriday.Node, entering bool) blackfriday.WalkStatus {
-			return mr.RenderNode(mr.buf, n, entering)
+			return mr.RenderNode(resBuf, node, entering)
```

Comment:
this fixes a bug when RenderNode was writing a buffer to itself and then reseting it

Reply:
(this made parts of source code randomly disappear, like if you use emphasis in a table cell)